settings.gradle

```diff
@@ -3,8 +3,9 @@ pluginManagement {
         eachPlugin {
             // Work around https://github.com/gradle/gradle/issues/1697.
             if (requested.version == null) {
+                def idName = requested.id.name.split('-').collect { it.capitalize() }.join().uncapitalize()
```

Comment:
Can you add an example to the commit message where this is needed? And maybe call the variable `pluginName` or `pluginId`?

Reply:
Done.